deepfence_server/model/scans.go

```diff
@@ -557,3 +564,9 @@ func (v CloudCompliance) GetCategory() string {
 func (CloudCompliance) GetJsonCategory() string {
 	return "severity"
 }
+
+type ScanComparison struct {
+	New     interface{} `json:"new" required:"true"`
+	Updated interface{} `json:"updated" required:"true"`
```

Comment:
Let's remove `Updated` entirely since it's not used anywhere.
If we need it, we can always add it later.

Reply:
cool. removing it!

Reply:
removed.